dockers/app-devel/Dockerfile

```diff
@@ -1,15 +1,16 @@
 FROM isard/alpine-pandas:latest
 MAINTAINER isard <info@isard.com>
 
-RUN apk add --no-cache git yarn py3-libvirt py3-paramiko py3-lxml py3-xmltodict py3-pexpect py3-openssl py3-bcrypt py3-gevent py3-flask py3-flask-login py3-netaddr py3-requests curl
+RUN apk add --no-cache bash yarn py3-libvirt py3-paramiko py3-lxml py3-pexpect py3-openssl py3-bcrypt py3-gevent py3-flask py3-netaddr py3-requests curl openssh-client
```

Comment:
```suggestion
RUN apk add --no-cache git bash yarn py3-libvirt py3-paramiko py3-lxml py3-pexpect py3-openssl py3-bcrypt py3-gevent py3-flask py3-netaddr py3-requests curl openssh-client
```

Reply:
git is not required anymore. Bash can be useful for debugging reasons, but I think it could be removed also.